Makefile

```diff
@@ -1,11 +1,11 @@
 .PHONY:	rpm clean
 
-VERSION ?= 3.4.14
+VERSION ?= 3.5.9
 BUILD_NUMBER ?= 1
-SOURCE = zookeeper-$(VERSION).tar.gz
+SOURCE = apache-zookeeper-$(VERSION)-bin.tar.gz
```

Comment:
The artifact name changed in version 3.5. Now it is prefixed with `apache-` and suffixed with `-bin`.